docs/tutorial/sandbox.md

```diff
@@ -0,0 +1,136 @@
+# Process Sandboxing
+
+One key security feature in Chromium is that processes can executed within a sandbox.
+The sandbox limits the harm that untrusted code can cause by limiting access to most
+system resources — sandboxed processes can only freely use CPU cycles and memory.
+In order to perform operations requiring additional privilege, sandboxed processes
+use dedicated communication channels to delegate tasks to more privileged processes.
+
+In Chromium, sandboxing is mainly applied to renderer processes. In addition, certain
+utility processes are also sandboxed, including the audio service, the GPU service,
+and the storage service. On Windows, the `chrome://sandbox` page will show you the exact
+sandboxing policy associated with each process.
+
+See Chromium's [Sandbox design document][sandbox] for more information.
+
+## Sandboxing in Electron
+
+Electron comes with a mixed sandbox environment, having Chromium's sandbox enabled but
+disabling it by default for renderer processes (meaning that it still applies to the
+relevant utility processes).
+
+Unsandboxed renderers are not a problem for desktop applications that only display
+trusted code, but it makes Electron less secure than Chromium for displaying untrusted
+web content. For applications that require more security, sandboxed renderers are
+recommended.
+
+Sandboxed processes in Electron behave _mostly_ in the same way as Chromium's do, but
+Electron has a few additional quirks to consider.
+
+### Renderer processes
+
+When renderer processes in Electron are sandboxed, they behave in the same way as a
+regular Chrome renderer would. A sandboxed renderer won't be initialized in a Node.js
+environment, nor will it modify any default JavaScript APIs that behave differently
+by default in Electron (e.g. `window.open`).
+
+<!-- TODO(erickzhao): when we have a solid guide for IPC, link it here -->
+Therefore, when the sandbox is enabled, renderer processes can only make changes to the
+system by delegating tasks to the main process via inter-process communication (IPC).
+
+### Preload scripts
+
+In order to allow renderer processes to communicate with the main process, preload
+scripts attached to sandboxed renderers will still have a limited Node.js environment
+enabled. Node's `require` module is exposed, but will only be able to import a subset
+of Electron and Node's built-in modules:
+
+* `electron` (only renderer process modules)
+* [`events`](https://nodejs.org/api/events.html)
+* [`timers`](https://nodejs.org/api/timers.html)
+* [`url`](https://nodejs.org/api/url.html)
+
+In addition, the preload script also exposes certain Node.js primitives as globals:
```

Comment:
```suggestion
In addition, the preload script also polyfills certain Node.js primitives as globals:
```

I want to emphasize that these are polyfills, not Node.js's implementation, they may not match the impl exactly and are provided as best effort implementations of the given APIs.